docs/01-overview/02-main-areas/application-connectivity/ac-09-useful-links.md

```diff
@@ -0,0 +1,48 @@
+---
+title: Application Connectivity - useful links
```

Comment:
Maybe the document could be placed at the bottom of the Application Connectivity section? I know it actually is the last doc regarding this component and that the next one is Runtime Agent; however, from the user's perspective, it's not.

Reply:
Hence the tile, to make it clear.
RA has its own list of useful links, AC docs end here, so I think it creates a nice division between these two components.
Just as the RA overview is not right after the AC overview.